test/test_node.py

```diff
@@ -12,16 +12,16 @@ def test_inputs_are_cut_to_match_arity():
 
     """
     idx = 0
```

Comment:
```suggestion
    node_idx = 0
```

Reply:
I think this is unrelated to this PR and the class member is called `idx` anyway.